docusaurus.config.js

```diff
@@ -5,20 +5,10 @@ module.exports = {
   tagline: 'Invincible applications, invisible infrastructure',
   url: 'https://docs.temporal.io',
   baseUrl: '/',
+  onBrokenLinks: 'warn',
```

Comment:
Why is this warn?

Reply:
> Is darkmode not default anymore?
> 
> Went and checked llinks seemed to not break anything
> 
> This came up in lighthouse. Not sure if its only because the link is for preview.
> 
> ![Screen Shot 2020-08-12 at 10 58 49 AM](https://user-images.githubusercontent.com/27736122/90050283-d677e280-dc8a-11ea-81dc-4574e271c320.png)

Dark mode was never explicitly set to be the default. I'm sending a new PR to set is as default (as per our offline discussion)

Indexing is disabled by Netlify because it's a preview